### DISCUSSION TITLE
AMLL 歌词审核员选拔 #1 （试行）（已结束）

### DISCUSSION THREAD

**Discussion by Steve-xmh** [amll-dev/applemusic-like-lyrics#223] [Announcements]
尽量完成以下歌曲的 TTML 歌词制作，通过率高且质量较高的将会选拔邀请进入 AMLL 歌词审核组。

届时可以获得便于审核的专用工具，一同协助维护和完善 AMLL 的歌词数据库哦！

根据情况可能会有多个人获得邀请，所以请多多努力吧！

## 选拔流程

请在下方评论区回复以表示自己参与本次选拔，并通过邮件方式发送你的 QQ 账户 ID 到 [stevexmh@qq.com](mailto:stevexmh@qq.com)，然后在从下方列出的曲目列表中选择自己喜欢的歌曲开始制作歌词即可。

歌词需要满足 [AMLL TTML DB 数据库 README](https://github.com/Steve-xmh/amll-ttml-db) 中列出的基本要求，优先审核要求可以在允许范围内提高质量。

如果列出的所有曲目均已完成且已经上传至数据库，则本次选拔结束。

如果您顺利得到了审核团的青睐，请注意你的邮箱是否收到邀请邮件回函，内附加入方式哦！

（同时也会在评论下回复您）

## 相关链接

- [AMLL TTML Tools 歌词制作工具 - 在线版](https://steve-xmh.github.io/amll-ttml-tool/)
- [AMLL TTML Tools 歌词制作工具 - 源码仓库](https://github.com/Steve-xmh/amll-ttml-tool)
- [AMLL TTML DB 歌词数据库（README 内含歌词提交要求供参考）](https://github.com/Steve-xmh/amll-ttml-db)
- [AMLL 本家仓库](https://github.com/Steve-xmh/applemusic-like-lyrics)

## 曲目列表

曲目以语言分类，以 `音乐ID 歌手 - 歌曲名` 格式列出，可以自行查询。

曲目可能会根据审核团的意愿动态增加（当然不会删除已有的曲目所以可以放心）。

同时可以利用 AMLL TTML Tools 提供的导入歌词数据库歌词功能，确认该歌曲没有 TTML 歌词，以免和他人冲突而浪费太多时间。

### 国语系列：
- ~~190449 张学友 - 吻别~~（已完成）
- ~~254141 梁静茹 - 暖暖~~（已完成）
- ~~349892 草蜢 - 失恋阵线联盟~~（已完成）
- ~~5285415 余天,李建复,洪荣宏,王梦麟,费玉清,齐秦,蔡琴,苏芮,潘越云,甄妮,林慧萍,王芷蕾,黄莺莺,陈淑桦,娃娃,李珮菁,齐豫,郑怡,江蕙,杨林 - 明天会更好~~（已完成）
- ~~480579325 梁咏琪,NekoLydia - 偏见~~（已完成）

### 粤语系列：
- ~~212233 陈慧娴 - 千千阕歌~~（已完成）
- ~~320950 叶丽仪 - 上海滩~~（已完成）
- ~~347230 Beyond - 海阔天空~~（已完成）

### 英语系列：
- 1377466954 Lucky Luke - Cooler Than Me
- 3986241 Carpenters - Yesterday Once More
- ~~406475394 AGA - Wonderful U (Demo Version)~~（已完成）

### 日语系列：
- 1992343682 RADWIMPS - カナタハルカ
- 2079797137 Ado - 唱
- 1382427349 福原遥 - 箱庭のサマー
- ~~2034657963 H△G - ふたりぼっち~~（已完成）
- 725259 fripSide - Heaven is a Place On Earth
- ~~4888621 Azuma,初音ミク,巡音ルカ - charActer~~（已完成）

**Comment by MianSoft** [comment]
已提交 `190449 张学友 - 吻别` https://github.com/Steve-xmh/amll-ttml-db/issues/576

**Comment by MianSoft** [comment]
已提交 `254141 梁静茹 - 暖暖` https://github.com/Steve-xmh/amll-ttml-db/issues/599

**Comment by MianSoft** [comment]
已提交 `349892 草蜢 - 失恋阵线联盟` https://github.com/Steve-xmh/amll-ttml-db/issues/606

**Comment by Miaoyww** [comment]
已提交 `406475394 AGA - Wonderful U (Demo Version)` https://github.com/Steve-xmh/amll-ttml-db/pull/853

**Comment by ench233** [comment]
已提交 `4888621 Azuma,初音ミク,巡音ルカ - charActer` [Steve-xmh/amll-ttml-db#917](https://github.com/Steve-xmh/amll-ttml-db/pull/917)

**Comment by ench233** [comment]
已提交 `320950 叶丽仪 - 上海滩` [Steve-xmh/amll-ttml-db#913](https://github.com/Steve-xmh/amll-ttml-db/pull/913)

**Comment by ench233** [comment]
已提交 `347230 Beyond - 海阔天空` [Steve-xmh/amll-ttml-db#908](https://github.com/Steve-xmh/amll-ttml-db/pull/908)

**Comment by ITManCHINA** [comment]
已提交`212233 陈慧娴 - 千千阕歌` https://github.com/Steve-xmh/amll-ttml-db/pull/1422
